src/apps/vpn/translations/strings.yaml

```diff
@@ -543,6 +543,24 @@ onboarding:
   panelThreeSubtitle: "Protecting yourself is simple: tap to connect to the VPN and you’re ready to browse safely. Mozilla VPN will start automatically for seamless protection."
   panelFourTitle: Add even more privacy
   panelFourSubtitle: You can set Mozilla VPN to protect you from ads, trackers, and malware. Plus, customize your protection for specific apps.
+  progressBarDataUse:
+    value: Data
+    comment: Label for onboarding progress bar step
+  progressBarMorePrivacy:
+    value: Privacy
+    comment: Label for onboarding progress bar step
+  progressBarAddDevices:
+    value: Devices
+    comment: Label for onboarding progress bar step
+  progressBarGetStarted:
+    value: Start
+    comment: Label for onboarding progress bar step
+  progressBarAccessibilityStepComplete:
+    value: "%1. Onboarding step %2 of %3. Select to go back to this step."
```

Comment:
Is %1 replace by Data, Privacy, etc. ?

Could we do something like this? `Onboarding step “%1” (%2 of %3). Select to go back to this step.`